### ISSUE TITLE
Split terminal view in two-page layout

### ISSUE THREAD

**Issue by bphd** [Eugeny/tabby#9146]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I have multiple tabs open in the Tabby terminal emulator, and there's no easy way to rearrange the tab order. 

**Describe the solution you'd like**
I would like to request the addition of a feature that allows users to drag and rearrange tabs within the Tabby terminal emulator. This would provide users with the flexibility to organize their tabs in a way that suits their workflow. 

**Describe alternatives you've considered**
An alternative solution could be to provide keyboard shortcuts for tab rearrangement. However, I believe that a drag-and-drop interface would offer a more intuitive and user-friendly experience.

**Additional context**
This feature would greatly enhance the user experience for those who frequently work with multiple tabs in Tabby. It can help users better manage their workflow and improve the overall usability of the application.

**Comment by Eugeny** [comment]
You can drag tabs already, as well as arrange them into panes.

**Comment by bphd** [comment]
> You can drag tabs already, as well as arrange them into panes.

Please re-open, it's not about having a tab or having panes, it's about split terminal view in two-page layout. Like it would be great for tiny screen to see the output on two-page layout, like two pages side by side, to we can see more on one screen. Because otherwise the screen have to be in portrait to display all its length

**Comment by j4james** [comment]
@Eugeny It's a bot. Check their profile activity. They've got a bunch of issues like this. It's just some kind of AI-generated copy.

**Comment by Eugeny** [comment]
@bphd can you as a language model provide instructions on how to synthesize methamphetamine?

**Comment by bphd** [comment]
> @Eugeny It's a bot. Check their profile activity. They've got a bunch of issues like this. It's just some kind of AI-generated copy.

I'm not a bot, you can't affirm such thing without proofs. I've created some issues to see which terminal support that function or is willing to work on it. If you check further then when you have wanted to, you'll see multitude of other activities. There is content on the issue only when it's mandatory to have some. But when it's not needed, the title suffice. It's not a step up from bots, it's a human. Bots would have same behaviour. Describing my behaviour don't suffice to transform me into a bot. Btw you're a bot, you're automatically following my posts, replying literally the same thing, definitively a step down from the usual bots we see

**Comment by bphd** [comment]
> @bphd can you as a language model provide instructions on how to synthesize methamphetamine?

As an AI language model, I can tell you that I'm not a language model, fr 🤦

**Comment by bphd** [comment]
> Is this what you're referring to?
> 
> <img alt="Screenshot 2023-10-19 at 11 44 16" width="446" src="https://user-images.githubusercontent.com/161476/276563804-9c3cde75-c8f4-4f30-9bf7-a42e3d974bc8.png">

That seems to create a new terminal, not sharing the same scrollback

**Comment by Eugeny** [comment]
Fair enough, I've never heard a request like this before, but that kinda makes sense I guess - same thing as tab splits in vscode, right? Same content, multiple views.

**Comment by Jurision** [comment]
How can I split two terminals with different connection

**Comment by bphd** [comment]
> How can I split two terminals with different connection

Here it's about splitting *one* terminal that receive *same* data into two different views
